### PR TITLE
No Bug - Clean up here-docs

### PR DESCRIPTION
**Pull request by dylanwh**
here-docs should either be at indent level 0 (and stored in constants)
or replaced with other forms of quoting that can be idented more readibly.

### CODE REVIEW

**Comment by dylanwh** [comment]
I want to add the caching headers for the static files, and I think here is the most sensible place. Before doing that I wanted to cleanup this a little bit.

File: Bugzilla/Install/Filesystem.pm [hunk 1]
```diff
@@ -49,6 +49,56 @@ use constant HT_DEFAULT_DENY => <<EOT;
 deny from all
 EOT
 
+use constant HT_GRAPHS_DIR => <<EOT;
```
Comment (floatingatoll) [inline Bugzilla/Install/Filesystem.pm]: Can this be a single q, so we can dump all the backslashing?

File: Bugzilla/Install/Filesystem.pm [hunk 2]
```diff
@@ -534,12 +534,8 @@ sub _remove_empty_css_files {
...
     my ($file) = @_;
     my $basename = basename($file);
-    my $empty_contents = <<EOT;
-/*
- * Custom rules for $basename.
- * The rules you put here override rules in that stylesheet.
- */
-EOT
+    my $empty_contents = qq{/* Custom rules for $basename.\n}
+        . qq{ * The rules you put here override rules in that stylesheet.  */};
```
Comment (floatingatoll) [inline Bugzilla/Install/Filesystem.pm]: Why don't we just make this comment one line instead of three?

File: Bugzilla/Install/Filesystem.pm [hunk 3]
```diff
@@ -481,14 +484,11 @@ sub update_filesystem {
...
-
-*** It appears that you still have an old index.html hanging around.
-    Either the contents of this file should be moved into a template and 
-    placed in the '$templatedir/en/custom' directory, or you should delete 
-    the file.
-
-EOT
+        print join("\n",
+            "*** It appears that you still have an old index.html hanging around.",
+            "Either the contents of this file should be moved into a template and ",
```
Comment (floatingatoll) [inline Bugzilla/Install/Filesystem.pm]: Trailing spaces here will be appended by newline, which works out to be a whitespace error that git can't see.

File: Bugzilla/Install/Filesystem.pm [hunk 4]
```diff
@@ -49,6 +49,56 @@ use constant HT_DEFAULT_DENY => <<EOT;
...
+<FilesMatch \\.(css|js)\$>
+  Allow from all
+</FilesMatch>
+
+# And no directory listings, either.
+Deny from all
+EOT
+
+use constant INDEX_HTML => <<EOT;
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```
Comment (floatingatoll) [inline Bugzilla/Install/Filesystem.pm]: Must we?

**Pull request closed without merging**